init.rb

```diff
@@ -12,12 +12,13 @@
 # Ignore the numerous backup files
 ignore(/~$/)
 
-# Verify that Ruby is NOT 1.8. 1.8 is unsupported
+
+# Ruby 1.8 is completly outdated, if you modify this, take respect to the addition checks below against 1.9 
 if defined?(RUBY_VERSION) && (RUBY_VERSION =~ /^1\.8\./)
```

Comment:
Looks like this section can be removed, since the next is checking on Ruby < 2.0